hpat/tests/test_series.py

```diff
@@ -1809,13 +1809,103 @@ def test_impl(S, param_skipna):
             result = hpat_func(S, param_skipna)
             self.assertEqual(result, result_ref)
 
-    def test_series_value_counts(self):
+    def test_series_value_counts_int(self):
```

Comment:
Looks like tests `test_series_value_counts_int` and `test_series_value_counts_float` could be combined, e.g.:
```
input_data = [test_global_input_data_integer64, test_global_input_data_float64]
extras = [[1, 2, 3, 1, 1, 3], [0.1, 0., 0.1, 0.1]]
for data, extra in zip(input_data, extras):
    for d in data:
        s = pd.Series(d + extra)
        # ...
```